contents/docs/libraries/ios/index.mdx

```diff
@@ -49,6 +49,15 @@ PostHogSDK.shared.alias("alias_id")
 
 We strongly recommend reading our docs on [alias](/docs/data/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) to best understand how to correctly use this method.
 
+## Person Profiles ("anonymous" vs "identified" persons)
+
+The default behavior for this SDK is to use the `identified_only` mode of person processing, but this can be changed in the config. The options are:
```

Comment:
```suggestion
The default behavior for this SDK is to use the `.identifiedOnly` mode of person processing, but this can be changed in the config. The options are:
```